shell/Library/bin/conda.bat

```diff
@@ -0,0 +1,47 @@
+@SET "_CONDA_EXE=%~dp0..\..\Scripts\conda.exe"
```

Comment:
Need to enclose `_CONDA_EXE` in quotes since `%~dp0` could contain spaces/other special characters.
Either here with `@SET "_CONDA_EXE='%~dp0..\..\Scripts\conda.exe"'` or (better?) when the variable is used.